.travis.yml

```diff
@@ -1,3 +1,6 @@
 language: ruby
 rvm:
-    - 2.2.1
+  - 2.2.1
+
+before_script:
+    - psql -c 'create database ebay_clone_test;' -U postgres
```

Comment:
This should just be `rake db:create`. If that doesn't work, you have something configured incorrectly.